src/components/Synthetics/PositionSeller/PositionSeller.scss

```diff
@@ -1,9 +1,5 @@
 .PositionSeller {
   overflow-x: hidden;
-
-  .Modal-body {
-    overflow-x: hidden;
```

Comment:
I can recall task about extra scroll in modals, can't this change bring such issue again?

Reply:
The overflow-x: hidden is a kind of a hack in this case. I had to remove it as it was a cause of issue while doing edge to edge background for the expandable group:
<img width="412" alt="Screenshot 2024-10-18 at 15 17 57" src="https://github.com/user-attachments/assets/0e2e012b-7d79-413e-b181-5a8bfbfda900">

We will need to find another solution if we encounter the scroll issue again